docs/doc/30-reference/30-sql/20-query-syntax/dml-select.md

```diff
@@ -20,6 +20,7 @@ SELECT
     [ORDER BY {col_name | expr | col_alias | col_position} [ASC | DESC], [ NULLS { FIRST | LAST }]
     [LIMIT row_count]
     [OFFSET row_count]
+    [FORMAT format]
```

Comment:
I prefer `IGNORE_RESULT` or `NO_OUTPUT` instead of reuse `FORMAT`

Reply:
+1, `FORMAT NULL` may be confused.

Reply:
What's the problem of reusing the Null as a `format` name?

Reply:
> What's the problem of reusing the Null as a format name?

It's a little confused.